src/core/include/openvino/op/squeeze.hpp

```diff
@@ -50,7 +50,7 @@ class OPENVINO_API Squeeze : public util::SqueezeBase {
     ///
     /// \param data Input tensor with data
     /// \param allow_axis_skip Shape inference result dynamic rank if selected axis has 1 in range of its dynamic
-    Squeeze(const Output<Node>& data, const bool allow_axis_skip = false);
+    Squeeze(const Output<Node>& data);
```

Comment:
It's public API and it should be deprecated first. Leave it as below and add deprecation notice of removal in 2026.0 release.
```suggestion
    Squeeze(const Output<Node>& data, bool);
```

Reply:
Or rather separate it into two ctors, one deprecated.